apps/orders/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from apps.common import models as base_models
+from apps.finance.models import Transaction
+from apps.inventory.models import Inventory
+from apps.users import models as usermodels
+
+
+class Order(base_models.BaseModel):
+    class Order_Status(models.TextChoices):
+        DRAFT = (
+            "D",
+            _("Draft"),
+        )
```

Comment:
This is just a suggestion... why is this not on one line like this `DRAFT = "D", _("Draft")`

Reply:
The isort cmd is refactoring it when I save it

Reply:
remove the outer brackets.. that should fix that